### ISSUE TITLE
Publish 2.11_3.0.2

### ISSUE THREAD

**Issue by eboto** [codingwell/scala-guice#53]
Can you publish one of the 3.0.x versions of this libraries for scala 2.11? Dependency issues are forcing some of us to stay on Guice 3 even as we upgrade to Scala 2.11.

**Comment by tsuckow** [comment]
I'll have to see. There was a compatibility issue that prevented it. It may be possible to backport the current 4.x code (or beta4) to 3.x

I don't have a ton of time as of late, so a pull request would probably speed it up.

**Comment by tsuckow** [comment]
I made some time tonight and it went much better than expected.
https://github.com/codingwell/scala-guice/tree/support/v3.0.3

I basically took the current 4.x series and undid the incompatible changes. The tests all pass, so it probably is fine. ¯\_(ツ)_/¯ Maybe @nbauernfeind could do a code review on the commit.

I pushed it as 3.0.3-SNAPSHOT, give it a spin and see if that works. I can see about making a release if so.
